src/lib/crypto/kimchi_bindings/pasta_fq_poseidon/kimchi_pasta_fq_poseidon.ml

```diff
@@ -2,5 +2,5 @@ type t
 
 external create : unit -> t = "caml_pasta_fq_poseidon_params_create"
 
-external block_cipher : t -> Kimchi.FieldVectors.Fq.t -> unit
+external block_cipher : t -> Kimchi_bindings.FieldVectors.Fq.t -> unit
```

Comment:
it'd be nice to use ocaml-gen for both this pasta_fp_poseidon and pasta_fq_poseidon packages